corehq/apps/es/aggregations.py

```diff
@@ -102,7 +102,7 @@ def buckets_dict(self):
 
     @property
     def buckets_list(self):
-        return {Bucket(b, self._aggregations) for b in self.normalized_buckets}
+        return [Bucket(b, self._aggregations) for b in self.normalized_buckets]
```

Comment:
@snopoke any reason you used a set originally? I added sorting to the aggregation and using a set here doesn't preserve the order.